scripts/main.js

```diff
@@ -1,5 +1,50 @@
 var helloCork = function() {
-  console.log('Hello Cork');
+  	console.log('Hello Cork');
 };
 
+var signedUrlUpcoming = "https://api.meetup.com/cork-javascript-meetup/events?photo-host=public&page=20&sig_id=210143296&sig=d07ac4460173bc86aa05e0c5b65acd79b2b8ae1b";
+var signedUrlPast = "https://api.meetup.com/cork-javascript-meetup/events?photo-host=public&page=20&sig_id=210143296&status=past&sig=052868f6f4d1f5a525fa6ca28a74e771165ad4ea";
+
+var getUpcomingEvents = function() {
+	var call = $.ajax({
+		type: "GET",
+		dataType: "jsonp",
+		url: signedUrlUpcoming
+	});
+	//console.log(call);
+	updateDOM(call, true);
+	return call;
+}
+
+var getPastEvents = function() {
+	var call = $.ajax({
+		type: "GET",
+		dataType: "jsonp",
+		url: signedUrlPast 
+	});
+	//console.log(call);
+	updateDOM(call, false);
```

Comment:
I think using the success handler would be better. It looks like your current solution would call the function in `ajaxStop` a total of four times instead of twice. You could use Something like this instead (and remove the ajaxStop):

```js
$.ajax({
  // ...
  success: updateDOM
});
```